Description: Drop doc package since it build fails and it is not really needed
Author: Andreas Tille <tille@debian.org>
Forwarded: not-needed
Last-Update: Thu, 24 Jan 2019 19:12:02 +0100

--- a/pom.xml
+++ b/pom.xml
@@ -104,6 +104,7 @@
         <impl_version>1.1.2</impl_version>
         <new_impl_version>1.2</new_impl_version>
         <non.final>false</non.final>
+        <maven.javadoc.skip>true</maven.javadoc.skip>
     </properties>
 
     <build>
